File: bench/lang/fib.py
import time

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)

start = time.time()
for _ in range(5):
    fib(28)
elapsed = time.time() - start
print("elapsed: %s" % elapsed)

File: bench/lang/fib.rb
def fib(n)
  return n if n < 2
  fib(n - 1) + fib(n - 2)
end

start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
5.times { fib(28) }
elapsed = Process.clock_gettime(Process::CLOCK_MONOTONIC) - start
puts "elapsed: #{elapsed}"
